base_station/gui/src/components/RoverMap.html

```diff
@@ -6,19 +6,30 @@
 </div>
 
 <script>
-  import L from 'leaflet';
-  import Marker from 'leaflet-rotatedmarker';
-
+  import L from '../leafletRover.js';
   export default {
     oncreate() {
-      const mymap = L.map(this.refs.map).setView([0, 0], 3);
-      L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}', {
-        attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, <a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="http://mapbox.com">Mapbox</a>',
+
+      const mymap = L.map(this.refs.map, {
+            center: [38.4062386734, -110.781233825],
+            zoom: 10,
+            minZoom: 1,
+            maxZoom: 18
+        }).setView([38.4062386734, -110.781233825], 10);
+      //L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}', {
```

Comment:
Remove these commented-out lines of code. We will be able to see the old code in git history.